server/methods/accounts/accounts.js

```diff
@@ -625,11 +630,13 @@ export function inviteShopOwner(options) {
 
   const emailLogo = Reaction.Email.getShopLogo(primaryShop);
   const token = Random.id();
-  const currentUser = Meteor.users.findOne(this.userId);
+  const currentUser = Meteor.user();
```

Comment:
I replaced this line here and below, which I _think_ is equivalent. I made this change to make the test easier, which I know is not a great reason to modify code, but on the plus side, it saves a db query.